app/models/waste_exemptions_engine/transient_registration.rb

```diff
@@ -28,24 +25,26 @@ class TransientRegistration < ActiveRecord::Base
     alias_attribute :people, :transient_people
     alias_attribute :registration_exemptions, :transient_registration_exemptions
 
-    def registration_attributes
-      attributes
-        .except(
-          "id", "token", "workflow_state", "start_option",
-          "declaration", "temp_operator_postcode",
-          "temp_contact_postcode", "temp_site_postcode",
-          "temp_grid_reference", "temp_site_description",
-          "address_finder_error", "transient_addresses",
-          "transient_registration_exemptions", "transient_people",
-          "created_at", "updated_at"
-        )
-    end
+    ATTRIBUTES_NOT_COPIED_TO_REGISTRATION = %w[address_finder_error
```

Comment:
What about calling these `TRANSIENT_ATTRIBUTES`? In general, I find a good practice to name something against what it *is* rather than against what it *isn't* :)

Reply:
Yeah, that's also a lot shorter 😅 I'll update it!

Reply:
Changed in https://github.com/DEFRA/waste-exemptions-engine/pull/138/commits/a3e4a617837f2c228d71e0e54c306ec9e4884ff0

Reply:
@cintamani Sorry, it voided your review!